src/components/calcite-inline-editable/calcite-inline-editable.tsx

```diff
@@ -154,10 +154,19 @@ export class CalciteInlineEditable {
   //
   //--------------------------------------------------------------------------
 
+  /**
+   * @todo doc
```

Comment:
who should add these?

Reply:
@alexabreu Can you take a stab at the doc for these events? I can update the PR if you can jot them in a comment.